RLTSquare/Ccq/Console/Command/CustomCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RLTSquare\Ccq\Console\Command;
+
+use Magento\Framework\MessageQueue\PublisherInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+
+class CustomCommand extends Command
+{
+    const TOPIC_NAME = 'rltsquare';
+    const Var1 = 'var1';
+    const Var2 = 'var2';
+    /**
+     * @var Json
+     */
+    protected Json $json;
+    /**
+     * @var PublisherInterface
+     */
+    protected PublisherInterface $publisher;
+    /**
+     * @var LoggerInterface
+     */
+    protected LoggerInterface $logger;
+
+    /**
+     * @param LoggerInterface $logger
+     * @param PublisherInterface $publisher
+     * @param Json $json
+     * @param string|null $name
+     */
+    public function __construct(
+        LoggerInterface $logger,
+        PublisherInterface $publisher,
+        Json $json,
+        string $name = null,
+    ) {
+        $this->publisher = $publisher;
+        $this->logger = $logger;
+        $this->json = $json;
+        parent::__construct($name);
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('rltsquare:hello:world');
+        $this->setDescription('Console command for adding a message into a queue');
+        $this->addArgument(
+            self::Var1,
+            null,
+            InputArgument::IS_ARRAY,
```

Comment:
Overlooked logic - The Symfony\Component\Console\Input\InputOption would be more suitable for task - see https://symfony.com/doc/current/console/input.html#using-command-arguments